[nepali_traditional] new layout for nepali

sapradhan opened the pull request:
Hi,
I wanted to contribute a layout I maintain for Nepali and ported to keyman.
The original layout is documented [here](https://nepalitankan.blogspot.com/2013/10/nepali-traditional.html)

Thanks

keyman-server commented:
This pull request is from an external repo and will not automatically be built. The build must still be passed before it can be merged. Ask one of the team members to make a manual build of this PR.

DavidLRowe commented:
Thanks for this contribution. Just a few comments:

The copyright statement should be:
```
(c) 2021 Santosh Pradhan
```
This change is needed in README.md, LICENSE.md, nepali_traditional.kmn and nepali_traditional.kps (Keyman package source) files.

The copyright statement is not required for the readme.htm or welcome.htm files, so you could remove the statement from those files, but if you keep the copyright statement you need to change it as above for README.md.

The recommended BCP 47 code for the language is "ne" (rather than "ne-NP") in nepali_traditional.keyboard_info file. Likewise the BCP 47 code in the nepali_traditional.kps (package source) file should be "ne" (rather than "ne-Deva-NP").

There are a few keys on each of the touch layouts that don't seem to be producing the right thing. (I'm not that familiar with Devanagari script, so I may be misunderstanding something, and I may well have missed other things.)

Touch layout (phone), default layer: 
- top row, fourth key, the two longpress keys seem to be swapped.
- top row, eighth key, third longpress key shows U+090A, but produces U+0917.
- third row, fourth key (counting the shift key), second longpress shows U+0905, but produces U+090B. 

numeric layer:
- top row, seventh key, longpress shows "7", but produces apostrophe (U+0027).
- third row, eighth key shows ";", but produces U+0938.
- third row, eighth key, first longpress key shows ":", but produces U+0938.
- third row, eighth key, second longpress key shows apostrophe, but produces U+0941.
- third row, eighth key, third longpress key shows double quote, but produces U+0942.
- third row, ninth key, shows "/", but produces U+0930.

Touch layout (tablet), default layer:
- third row, tenth key, first longpress key shows U+0940, but produces U+0941.
- (optional) fourth row, ninth key, longpress is the same as the base (both produce comma).

symbol layer:
- (optional) You might consider moving the longpress keys to some of the empty keys on this layer.

sapradhan commented:
Thank you for thoroughly reviewing my submission. 

> 
> 
> The copyright statement should be:
> 
> ```
> (c) 2021 Santosh Pradhan
> ```
> 
> This change is needed in README.md, LICENSE.md, nepali_traditional.kmn and nepali_traditional.kps (Keyman package source) files.
> 
> The copyright statement is not required for the readme.htm or welcome.htm files, so you could remove the statement from those files, but if you keep the copyright statement you need to change it as above for README.md.
> 
Done.

> The recommended BCP 47 code for the language is "ne" (rather than "ne-NP") in nepali_traditional.keyboard_info file. Likewise the BCP 47 code in the nepali_traditional.kps (package source) file should be "ne" (rather than "ne-Deva-NP").
> 
Done.


> There are a few keys on each of the touch layouts that don't seem to be producing the right thing. (I'm not that familiar with Devanagari script, so I may be misunderstanding something, and I may well have missed other things.)
> 
> Touch layout (phone), default layer:
> 
>     * top row, fourth key, the two longpress keys seem to be swapped.
> 
>     * top row, eighth key, third longpress key shows U+090A, but produces U+0917.
> 
>     * third row, fourth key (counting the shift key), second longpress shows U+0905, but produces U+090B.
> 
> 
> numeric layer:
> 
>     * top row, seventh key, longpress shows "7", but produces apostrophe (U+0027).
> 
>     * third row, eighth key shows ";", but produces U+0938.
> 
>     * third row, eighth key, first longpress key shows ":", but produces U+0938.
> 
>     * third row, eighth key, second longpress key shows apostrophe, but produces U+0941.
> 
>     * third row, eighth key, third longpress key shows double quote, but produces U+0942.
> 
>     * third row, ninth key, shows "/", but produces U+0930.
> 
I had misunderstood K_xxx would produce that character. Now fixed


> Touch layout (tablet), default layer:
> 
>     * third row, tenth key, first longpress key shows U+0940, but produces U+0941.
> 
>     * (optional) fourth row, ninth key, longpress is the same as the base (both produce comma).
> 
> 
> symbol layer:
> 
>     * (optional) You might consider moving the longpress keys to some of the empty keys on this layer.

Addressed all these changes and reorganized a few keys. I apologize for not testing things properly before submitting, this time I tested the touch layout on a browser. Please let me know if I need to make any more changes.
